swift/extractor/trap/TrapOutput.h

```diff
@@ -39,6 +39,12 @@ class TrapOutput {
     print(e);
   }
 
+  template <typename... Args>
```

Comment:
maybe we should leave a TODO here to make this conditional on a flag or environment variable?

Reply:
So far it's only a way to emit debug info while working locally, not something we call unconditionally (at least I did not intend to actually use it in this PR).

Reply:
ok I see! we probably should be having some more permanent trap debugging tools, but we can do it at a later stage